### PR TITLE
Pulp vhosts80 moved to conf.d

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by ekohl** [comment]
Requires https://github.com/theforeman/puppet-pulp/pull/350

File: templates/pulp-apache.conf.erb [hunk 1]
```diff
@@ -10,7 +10,7 @@ Alias /pub /var/www/html/pub
   Require all granted
 </Location>
 
-Include /etc/pulp/vhosts80/*.conf
+Include /etc/httpd/conf.d/vhosts80/*.conf
```
Comment (evgeni) [inline templates/pulp-apache.conf.erb]: `puppet-pulp` uses `${apache::confd_dir}`, but here we hardcode `/etc/httpd/conf.d`? (Debian ships httpd config in `/etc/apache2` ;))

**Comment by evgeni** [comment]
Failures seem unrelated?

File: templates/pulp-apache.conf.erb (hunk 1 above)
Comment (evgeni) [reply templates/pulp-apache.conf.erb]: this comment remains

**Comment by ehelms** [comment]
Updated @evgeni

File: templates/pulp-apache.conf.erb [hunk 2]
```diff
@@ -10,7 +10,7 @@ Alias /pub /var/www/html/pub
   Require all granted
 </Location>
 
-Include /etc/pulp/vhosts80/*.conf
+Include <%= scope['apache::confd_dir'] %>pulp-vhosts80/*.conf
```
Comment (evgeni) [inline templates/pulp-apache.conf.erb]: I'm not sure whether `apache::confd_dir` does or does not contain a trailing slash, so I'd prefer to be explicit here and add one before `pulp-vhost80`.

Comment (evgeni) [reply templates/pulp-apache.conf.erb]: https://github.com/puppetlabs/puppetlabs-apache/blob/master/manifests/params.pp#L72 says it doesn't.

**Pull request merged**